internal/cli/install.go

```diff
@@ -375,9 +367,17 @@ func (c *InstallCommand) Run(args []string) int {
 	s.Update("Server installed and configured!")
 	s.Done()
 
+	// NOTE(briancain): Right now we rely on the fact that only K8S returns a bootstrap
+	// token from the installation interface helper. Since we know helm sets up
+	// a bootstrap job AS WELL AS a runner, we don't need to install another runner
+	// or another runner profile.
 	if c.flagRunner {
+		existingODRConfigSetup := false
+		if bootstrapToken != "" {
```

Comment:
I don't love this conditional being based on the `bootstrapToken` value being returned from the installer interface. We could add a fourth return value which is a bool that says whether or not it already set up a runner profile? I was trying to keep this change small so went with this instead but if people feel strongly that relying on `bootstrapToken` being set is bad I can add another return bool param.